README.md

```diff
@@ -1,26 +1,40 @@
-business-days-calculator
-========================
+# business-days-calculator
 
 Business Days Calculator
 
-
 [![Build Status](https://travis-ci.org/andrejsstepanovs/business-days-calculator.png?branch=master)](https://travis-ci.org/andrejsstepanovs/business-days-calculator) [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/andrejsstepanovs/business-days-calculator/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/andrejsstepanovs/business-days-calculator/)[![Coverage Status](https://coveralls.io/repos/andrejsstepanovs/business-days-calculator/badge.png?branch=master)](https://coveralls.io/r/andrejsstepanovs/business-days-calculator?branch=master) [![Latest Stable Version](https://poser.pugx.org/andrejsstepanovs/business-days-calculator/v/stable.png)](https://packagist.org/packages/andrejsstepanovs/business-days-calculator) [![License](https://poser.pugx.org/andrejsstepanovs/business-days-calculator/license.png)](https://packagist.org/packages/andrejsstepanovs/business-days-calculator)
 
 ## Install
 
 * If you're using Composer to manage dependencies, you can use
 
 ```sh
-composer require andrejsstepanovs/business-days-calculator
+composer require stockx/business-days-calculator
 ```
 
 or add to your composer.json file:
 
-    "require": {
-        "andrejsstepanovs/business-days-calculator": "1.*",
-    }
+```json
+"require": {
+    "stockx/business-days-calculator": "1.*",
+}
+```
+
+## Releasing a New Version
+
+This process takes a few steps, both in this repository and (typically) in Grails.
+
+Create a branch and make your changes here.  You can label that branch with a semver convention and a stability at the end.  For example `git checkout -b feature-name-dev` will match a `composer.json` requirement value of `dev-feature-name` in consuming projects.  For this to actually work the `composer.json` in this project will need to have its `version` tag updated to match `dev-feature-name`.  Then, this can be used to test with Composer locally on Grails.  **NOTE:** Actual semver numbers should not be used to in local testing.
```

Comment:
I think I'd like this broken out into an ordered list of: this is what you do for local testing, this is what you do for a full release.

Reply:
Alright @emmercm, reworked.  Lemme know what you think.